### PR TITLE
add proto stub

### PR DESCRIPTION
**Pull request by domfoy**
Stub since there are clearly not all the server actions.

### CODE REVIEW

File: src/protos/stub.proto [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+message Container {
+  optional PlayerAction player_action;
+  optional ServerAction server_action;
+  optional SpectatorAction spectator_action;
+}
+
+message Game {
+  optional uint32 id;
+  repeated Player players;
+  repeated Card cards;
```
Comment (destiny06) [inline src/protos/stub.proto]: I'm not clear on this one: I was thinking of having a list of cards for the discarded pile, list of cards to pick, and a list of cards on the board ?
what is your list representing ?

File: src/protos/stub.proto [hunk 2]
```diff
@@ -0,0 +1,174 @@
+package bicyclade;
```
Comment (destiny06) [inline src/protos/stub.proto:7]: in proto we can include others prots: import "myproject/base.proto"
we may consider it bt can be done in another PR

File: src/protos/stub.proto [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+  optional ServerAction server_action;
+  optional SpectatorAction spectator_action;
+}
+
+message Game {
+  optional uint32 id;
+  repeated Player players;
+  repeated Card cards;
+  repeated Offering offerings;
+  repeated GodType gods;
```
Comment (destiny06) [inline src/protos/stub.proto]: a god has a position in the godlist no ? not only godType

File: src/protos/stub.proto (hunk 1 above)
Comment (domfoy) [reply src/protos/stub.proto]: Yes I forgot the discarded cards. This list is supposed to represent the cards on the board. I don't think the cards to pick should be communicated to the clients.

File: src/protos/stub.proto (hunk 3 above)
Comment (domfoy) [reply src/protos/stub.proto]: Agreed

**Pull request closed without merging**